reference/configuration/tidb-server/tidb-specific-variables.md

```diff
@@ -43,6 +43,13 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join.
 - When the aggregate operation is slow in query, you can set the variable value to 1.
 
+### tidb_opt_distinct_agg_push_down
+
+- Scope: SESSION
+- Default value: 0
+- This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function with distinct(like `select count(distinct a) from t`) to Coprocessor.
+- When the aggregate with distinct operation is slow in query, you can set the variable value to 1.
```

Comment:
please add an example to describe how to confirm that the `distinct` is pushed down.